crates/bevy_pbr/src/render/light.rs

```diff
@@ -653,7 +653,7 @@ pub fn calculate_cluster_factors(
 // we will also construct it in the fragment shader and need our implementations to match,
 // so we reproduce it here to avoid a mismatch if glam changes. we also switch the handedness
 // could move this onto transform but it's pretty niche
-pub(crate) fn spot_light_view_matrix(transform: &GlobalTransform) -> Mat4 {
+pub(crate) fn spot_light_view_from_world(transform: &GlobalTransform) -> Mat4 {
```

Comment:
I feel like this is maybe world from view.

Reply:
Yep, it's being built from `GlobalTransform` too, which should've been a hint. :sweat_smile: